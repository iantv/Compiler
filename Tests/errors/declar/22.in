int (*a[10])() typedef