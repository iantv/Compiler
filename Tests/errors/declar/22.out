error C2143: missing ";" before "typedef" in line 1 column 16
